sodafoundation/delfin
# Adding Prometheus alert manager exporter

**What this PR does / why we need it**:
New alert exporter to prometheus alert manager
recieved alert in delfin ( through trap and sync  ) will be pushed to alert manager through this exporter.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #NA

**Special notes for your reviewer**:
This is rwork of PR https://github.com/sodafoundation/delfin/pull/429
**Release note**:

1. Exporter for Prometheus alert manager

```release-note
```


**Test Report**:
1. Sync alerts for storage <storage_id>
![image](https://user-images.githubusercontent.com/45681499/102973671-2c0aa880-4523-11eb-86a8-7688f36d8b92.png)

2. Check task process logs
![image](https://user-images.githubusercontent.com/45681499/102973718-45135980-4523-11eb-9b30-16383db50e2d.png)

3. Check alert manager for alerts
![image](https://user-images.githubusercontent.com/45681499/102973784-63795500-4523-11eb-9e10-2942d0af44a5.png)
